docs/ert/getting_started/howto/design_matrix.rst

```diff
@@ -10,7 +10,7 @@ contains parameters and their values per realization.
 parameters as `DESIGN2PARAMS`.
 
 If the same parameter name is defined both in the design matrix and in the GEN_KW parameter definition,
-the value from the design matrix takes precedence and the parameter will not be updated by ERT.
+the priority of which values are to be used is defined by optional attribute PRIORITY:<design_matrix|sampled>.
```

Comment:
It feels confusing that when choosing between "GEN_KW" and "DESIGN_MATRIX" you have to choose between design_matrix and sampled. "sampled" is more a property than a "source".